spec/requests/post_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /index' do
+    before :each do
+      get '/users/1/posts'
+    end
+    it 'returns success status' do
+      expect(response).to have_http_status(:success)
+    end
```

Comment:
Great job @78fatemanazari 🤝
- You are doing a great job on implementing the tests. However, this test case is still failing the check. Kindly fix it and ensure it passes the test check.
![error](https://github.com/78fatemanazari/Interacti_Blog/assets/63917725/a1537a64-6c03-445f-b2b9-9b3bcb44c35e)

Reply:
Thanks a lot, @1GORDON for your review, but all the tests are passed successfully inside my local machine.
Please check the screenshot below.
![test](https://github.com/78fatemanazari/Interacti_Blog/assets/119149574/461f6ffe-fce5-47f4-95cf-7f895abf2f80)